src/envo/envo-edit.owl

```diff
@@ -38519,6 +38520,15 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000132> "f
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000132> <http://purl.obolibrary.org/obo/ENVO_00000444>)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000132> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002430> <http://purl.obolibrary.org/obo/ENVO_01001292>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000133> (agricultural potting mixture)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Potting_soil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000133> "An agricultural environmental material, which comprises a mixture of materials such as peat, composted bark, sand, perlite, or compost, and serves as a medium by which to grow plants.")
```

Comment:
It looks like the DbXref is in a string rather than IRI format - please make sure you're using the IRI editor when adding these (there are quite a few we need to clean up in the whole ontology, legacy from when I didn't know better) 

Also, the definition needs some cleanup (no comma before which, avoid "comprise" and "various", have to be a bit more precise on what it actually is, etc)

We can keep this under AEM for now, but we'll need a better hierarchy eventually. 

>Definition: An agricultural environmental material which is formulated to support the growth of plants in a pot or other durable container.

>Comment: Potting soil may be composed of a wide range of materials in very different ratios, and typically contains little or no actual soil. Materials found in potting soil often include peat, composted bark, sand, perlite, or compost.

Reply:
> It looks like the DbXref is in a string rather than IRI format - please make sure you're using the IRI editor when adding these

@pbuttigieg  I don't know what you mean by the IRI editor. I had added the link as an db xref annotation property as I understood you had taught me to do.  Can you be more specific or show me what you mean by the `IRI editor`?

Reply:
> no comma before which

@pbuttigieg I was taught in my graduate writing course that in American English in this context one must always but a comma before "which". I understand that you prefer British English but both are correct and frankly I don't see the point in being that pedantic about it. From my observation most OBO ontologies use "that" instead of "which".

Reply:
>@pbuttigieg I don't know what you mean by the IRI editor. I had added the link as an db xref annotation property as I understood you had taught me to do. Can you be more specific or show me what you mean by the IRI editor?

Right now, the dbxref is stored as a "literal" string....
![image](https://user-images.githubusercontent.com/2276146/87426329-48eb9900-c5df-11ea-921d-5c1e1a883d64.png)

We want to cast it as an IRI format by entering the URL in the "IRI Editor" tab like so...

![image](https://user-images.githubusercontent.com/2276146/87426437-720c2980-c5df-11ea-8e5c-8ce57b01d6c2.png)

> From my observation most OBO ontologies use "that" instead of "which".

This is a matter of preference, as you used "which" I corrected your grammar. You're free to use that if you wish.